meetings/2019-12-17.md

```diff
@@ -12,16 +23,106 @@
+      - The ISIS TC has complete control over policy documents like the contributing document.
+
+  - [TC Charter section 4](https://github.com/USGS-Astrogeology/ISIS_TC/issues/63)
+    - Ross proposed a change here.
+        - There was no response besides some thumbs up emojis.
+        - Ross should submit a PR for this change.
+
+  - [TC Charter section 5](https://github.com/USGS-Astrogeology/ISIS_TC/issues/64)
+    - There is still uncertainty about what technical direction is.
+      - We should continue this discussion asynchronously and try to have a solid definition.
+    - **Jesse** - The ISIS TC taking on all of these listed responsibilities immediately is intractable. We should gradually move responsibilities from the ASC to the ISIS TC.
+      - We need a way to denote which responsibilities are currently being done by the ASC and which are being done by the ISIS TC.
+        - The person that writes the PR for this can decide how to do this.
+        - We should not actually hide the responsibilities if the ISIS TC is not doing them right now.
+  - Should we combine all of these changes into one big PR?
+    - It is generally better to use multiple isolated PRs for this.
+    - Goal is to have all PRs merged 1 week out of next week so people have time to review the charter before final voting.
 
-  - Dropping GCC4 Support
-    - [ISIS issue](https://github.com/USGS-Astrogeology/ISIS3/issues/3566)
+  - [Dropping GCC4 Support](https://github.com/USGS-Astrogeology/ISIS3/issues/3566)
+    - ISIS is currently compiled and tested on Ubuntu 18, Fedora 28, and CentOS 7.
+      - Ubuntu and Fedora use GCC7
+      - CentOS 7 uses GCC4
+      - Binaries compiled with GCC7 are not compatible with GCC4.
+      - Conda-forge, the main provider of ISIS3 dependencies, recently switched from GCC4 to GCC7.
+    - The ASC currently provides special builds of some third party dependencies that are compatible with GCC4.
+      - Because conda-forge changed, the ASC would have to provide GCC4 compatible versions of third party dependencies if we move to new versions.
+      - The ASC does not want to be providing third party dependencies.
+        - the ASC has work on the books right now to reduce the number of third party dependencies they serve.
+    - For these reasons the ASC would like to stop supporting compiling with GCC4.
+      - ISIS can still be installed and run on CentOS7, or any other system that uses GCC4, but it could not be built from source.
+        - When multiple OSs were supported with the same build in the past, there were small problems that cropped up.
+        - OSIRIS-REx currently compiles on CentOS 7 and would have difficulty moving away from it.
+    - The ASC doesn't need to ask the ISIS TC if they should do this.
+      - This discussion is very related to the previous discussion about how much the ISIS TC can direct contributors.
```

Comment:
We as the TC can make a recommendation, but contributors may not have to agree to follow through with them (hypothetically we could recommend the ASC to continue GCC4 support for 3rd parties despite external directives) right?

Reply:
I think it would be more: "GCC4 support for 3rd parties should be supported", but yes.